functions-and-scope.js

```diff
@@ -52,27 +82,49 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade(grades) geeft 6.642857142857143
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
-
-
+console.log("\n   Opdracht  2: Gemiddeld cijfer, 2b: Omschrijven tot een herbruikbare functie")
+const averageGrade = arrayGrades => {
+    let totalOfGrades = 0
+    for (i in arrayGrades) {
+        totalOfGrades = totalOfGrades + arrayGrades[i]
+    }
+    return "Gemiddelde cijfer: " + totalOfGrades / arrayGrades.length + "   Cijfers: " + arrayGrades
+}
+console.log(averageGrade(grades))
+console.log(averageGrade([6, 4, 5]))
+console.log(averageGrade([8, 9, 4, 6, 10]))
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
-
-
-
-
+console.log("\n   Opdracht  2: Gemiddeld cijfer, 2c: Afronden op twee decimalen")
+const averageGrade2Decimals = arrayGrades => {
+    let totalOfGrades = 0
+    for (i in arrayGrades) {
+        totalOfGrades = totalOfGrades + arrayGrades[i]
+    }
+    return "Gemiddelde cijfer: " + Math.round((totalOfGrades / arrayGrades.length)*100)/100 + "   Cijfers: " + arrayGrades
```

Comment:
Wist je dat je ook de .toFixed(2) kon gebruiken?
```suggestion
    return "Gemiddelde cijfer: " + (totalOfGrades / arrayGrades.length).toFixed(2) + "   Cijfers: " + arrayGrades
```